### PR TITLE
Add parole/vocabulary for capitolo-01

### PR DESCRIPTION
**Pull request by matt5115**
I created my own branch.  I added the parole.md for ch1 and ch 2. My first time.  I am not sure if I did this properly.

### CODE REVIEW

**Comment by chrstphfrtz** [comment]
Yes, that worked perfectly. The question is: do we need the vocabulary or is there a specific reason we should add them? I guess it would make sense to add translations if people want to print out a page for studying?

**Comment by matt5115** [comment]
Yes. That was my thought.  To add translations.  Basically to create one place with all the resources for the book.

**Comment by chrstphfrtz** [comment]
I see. That makes sense. Could you please create a separate branch + pull request for every topic? Then we can review the changes in smaller chunks.

So basically you need to create two new pull requests. One containing the words from Capitolo 02 and one containing the exercises from Capitolo 02. This one we can use for the words from Capitolo 01.

If you need help with this just let me know!

**Comment by matt5115** [comment]
Yes, I will create the separate branch + pull request for every topic.  Sorry, I am new to github

File: capitolo-01/parole.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+bambino,-i
```
Comment (chrstphfrtz) [inline capitolo-01/parole.md:1]: Please format the words like here https://github.com/chrstphfrtz/litaliano-secondo-il-metodo-natura/blob/capitolo-03-parole/capitolo-03/parole-it-en.md

This is what the format looks like in markdown: https://raw.githubusercontent.com/chrstphfrtz/litaliano-secondo-il-metodo-natura/capitolo-03-parole/capitolo-03/parole-it-en.md

The maximum line count per page is 45 so please continue in the 3rd column after line 45.

**Review by chrstphfrtz — CHANGES_REQUESTED** [review]

File: capitolo-01/parole.md (hunk 1 above)
Comment (matt5115) [reply capitolo-01/parole.md:1]: OK, I will work on this.

**Pull request closed without merging**